examples/convert_group_studies.py

```diff
@@ -61,6 +62,11 @@
 ###############################################################################
 # Let us loop over the subjects and create BIDS-compatible folder
 
+daysback_min, daysback_max = get_anonymization_daysback(
+    [mne.io.read_raw_fif(op.join(data_path, repo, 'sub%03d' % subject_id,
+                                 'MEG', 'run_%02d_raw.fif' % run))
+     for run in runs for subject_id in subject_ids])
```

Comment:
I think this would be cleaner as a for loop like:

```Python
mylist = list()
for sub in subjs:
    for run in runs:
        raw = mne.io.read_.......
        mylist.append(raw)

daysbackmin, daysbackmax = get_anon(mylist
```

more verbose ... but also much more obvious. Since we are dealing with an example here, I think it's good to be obvious.

Also: Add a short comment on top of these lines, explaining what you are doing / why

Reply:
keep this private and let the error message say what to do. We should fight to keep the public API as small as possible to avoid future maintenance cost.